src/components/Typography/index.tsx

```diff
@@ -13,24 +11,51 @@ import {
 import { FiArrowUpRight } from "react-icons/all"
 import { ExternalLinkHref } from "../Navbar/types"
 
-export const Headline: FC<HeadingProps> = (props) => {
-  return <Heading fontSize="7xl" {...props} />
-}
-
 export const H1: FC<TextProps> = (props) => {
-  return <Text as="h1" fontSize="6xl" lineHeight="64px" {...props} />
+  return (
+    <Text
+      as="h1"
+      fontSize={{ base: "48px", md: "60px" }}
+      lineHeight={{ base: "52px", md: "64px" }}
```

Comment:
I've noticed that it probably won't work with cms. Eg. we want to add custom content for a page and we can use  `<h1-6>` tags. In that case, the `<h1>` tag from the cms data will be different from the `H1` react component. It's hard to explain 😆, so let's see an example.

- [this content](https://github.com/keep-network/website/blob/master/src/pages/terms-of-use.md?plain=1#L7) we can create via cms and we can't use `H2` react component- it will render `h2` w/o styles provided in `H2` react component,
- ^ content will be rendered here https://github.com/keep-network/website/blob/master/src/templates/legal-page.js#L41 https://github.com/keep-network/website/blob/master/src/templates/legal-page.js#L19.

So I believe we should override the chakra **global styles** so even if we use `<h2>` or `<H2> as react component` the styles will be the same.

Reply:
@r-czajkowski  what about using individual props for each element instead of including the entire body as one massive prop?

Reply:
The individual props for each item are fine and we're going to use this pattern as well, but I think we should use it for components where they have the template eg. an [`SplashCard`](https://github.com/threshold-network/splash-page/blob/gatsby/src/components/SplashCard.tsx) or [navigation items](https://github.com/threshold-network/splash-page/blob/gatsby/src/components/Navbar/index.tsx#L12-L35).

Let's see an announcement banner component https://github.com/threshold-network/splash-page/blob/gatsby/src/components/Navbar/WhatsNextBanner.tsx#L34. If we decide to individual prop for each element we would have to agree on a template eg.: `<text> <external link at the end of the banner>`- this limits us to a fixed pattern. We don't have the flexibility eg. we can't put 2 external links, at the beginning and at the end of the banner, because we are limited by a React component. Markdown parser + netlify cms give us more power in some cases.

Reply:
We are going to address the global styles issues in the follow-up work.